capellambse/model/crosslayer/capellacore.py

```diff
@@ -26,8 +27,163 @@ class Generalization(c.GenericElement):
     _xmltag = "ownedGeneralizations"
 
 
+@c.xtype_handler(None)
+class EnumerationPropertyLiteral(c.GenericElement):
+    """A Literal for EnumerationPropertyType."""
+
+    _xmltag = "ownedLiterals"
+
+
+@c.xtype_handler(None)
+class EnumerationPropertyType(c.GenericElement):
+    """An EnumerationPropertyType."""
+
+    _xmltag = "ownedEnumerationPropertyTypes"
+
+    literals = c.DirectProxyAccessor(
+        EnumerationPropertyLiteral, aslist=c.ElementList
+    )
+
+
+class PropertyValue(c.GenericElement):
+    """Abstract base class for PropertyValues."""
+
+    _xmltag = "ownedPropertyValues"
+
```

Comment:
~Do we want a reference back to its group? This is only needed if one is working with `PropertyValue`s via `search` for example.~

```suggestion

group = c.ParentAccessor(c.GenericElement)
```

Reply:
**Rejected** since this leads to ambiguous returns. Example `StringPropertyValue`'s group can lead to a `EnumerationPropertyLiteral` instead of a `PropertyValueGroup`.